### PR TITLE
Implemented classes Medic and Examination. Added some logic to the MedicController class.  -- 15.01. - Created an application package with the main method. Added view for Medic branch testing and implemented some logic in Controller and View. -- 17.01. Continued work, updated MedicController

### PR DESCRIPTION
**Pull request by ghost**
Please review and feedback me!

### CODE REVIEW

**Review by deutschha — APPROVED** [review]
looks good 👍

**Comment by ghost** [comment]
Did some work on the code today. Please review the new code as well. Thanks!

**Comment by ghost** [comment]
Continued work on branch. Updated MedicController and added some methods.

File: src/main/java/model/persons/Address.java [hunk 1]
```diff
@@ -44,4 +44,9 @@ public String getStreet() {
     public void setStreet(String street) {
         this.street = street;
     }
+
+    @Override
+    public String toString() {
+        return street+" "+houseNumber+", "+postcode+" "+city;
```
Comment (x21L) [inline src/main/java/model/persons/Address.java:50]: please reformat code next time. `" + postcode + "`

**Review by x21L — APPROVED** [review]

File: src/main/java/model/persons/Address.java (hunk 1 above)
Comment (ghost) [reply src/main/java/model/persons/Address.java:50]: Okay - thanks!

**Pull request merged**